_posts/2022-03-22-publiccode-yaml-analyse.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title: Open Source Code öffentlicher Verwaltungen am Beispiel Italien
+author: Thomas Werner
+category: blog
+twitter: toms_rocket
+keywords:
+  - "Open Source"
+  - "Italien"
+  - "Public Code"
+---
+
+In Italien gibt es einen Metadaten-Standard für Software, die von der öffentlichen Verwaltung entwickelt oder erworben wurde. Und zwar besagt der Standard, dass in jedem GIT(Hub)-Repository, das entsprechenden öffentlich geförderten Quellcode enthält, eine Datei mit dem Namen "publiccode.yml" liegen soll. Diese Datei enthält verschiedene Informationen über das Repository bzw. über die enthaltene Software.
```

Comment:
Etwas holpriger Einstieg.

Vielleicht besser?

"Die Italienische Regierung (??) entwickelte 1986 (??) den Metadaten-Standard "publiccode.yml" mit dem Ziel X und Y. Üblicherweise wird diese "publiccode.yml" zusammen mit dem Code veröffentlicht und erlaubt es uns einen kleinen Einblick in die in Italien entwickelte Software zu werfen."

Reply:
Gute Idee, aber sorry, ich kriege gerade keine vernünftige Neuformulierung hin und habe keinen Bock mehr da noch weiter dran zu feilen.